File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/spot/SpotAccountApi.java
package com.bitget.openapi.api.spot;

import com.bitget.openapi.dto.request.spot.SpotBillQueryReq;
import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.Query;

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: spot account api
 */
public interface SpotAccountApi {

    /**
     * Obtain account assets
     * @return ResponseResult
     */
    @GET("/api/spot/v1/account/assets")
    Call<ResponseResult> assets();

    /**
     * Get the bill flow
     * @param spotBillQueryReq
     * @return ResponseResult
     */
    @POST("/api/spot/v1/account/bills")
    Call<ResponseResult> bills(@Body SpotBillQueryReq spotBillQueryReq);

    /**
     * Obtain transfer records
     * @param coinId
     * @param fromType
     * @param limit
     * @param after
     * @param before
     * @return ResponseResult
     */
    @GET("/api/spot/v1/account/transferRecords")
    Call<ResponseResult> transferRecords(@Query("coinId") String coinId,
                                         @Query("fromType") String fromType,
                                         @Query("limit") String limit,
                                         @Query("after") String after,
                                         @Query("before") String before);
}

File: bitget-php-sdk-api/src/api/spot/SpotAccountApi.php
<?php


namespace bitget\api\spot;


use bitget\internal\BitgetApiClient;
use bitget\model\spot\account\BillsReq;

class SpotAccountApi
{
    const BASE_URL = "/api/spot/v1/account";

    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }
    /**
     * Obtain account assets
     * @return string
     */
    public function assets(): string
    {
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/assets", null);
    }
    /**
     * Get the bill flow
     * @param $billsReq
     * @return string
     */
    public function bills(BillsReq $billsReq): string
    {
        return $this->BitgetApiClient->doPost(self::BASE_URL . "/bills", $billsReq);
    }
    /**
     * Obtain transfer records
     * @param $coinId
     * @param $fromType
     * @param $limit
     * @param $after
     * @param $before
     * @return string
     */
    public function transferRecords(string $coinId, string $fromType,string $limit,string $after,string $before): string
    {
        $params = array("coinId" => $coinId, "fromType" => $fromType, "limit" => $limit, "after" => $after, "before" => $before);
        return $this->BitgetApiClient->doGet(self::BASE_URL . "/transferRecords", $params);
    }


}